docs/ig.md

```diff
@@ -1,25 +1,25 @@
 ---
-title: local-gadget
+title: IG
 weight: 80
 description: >
-  Description of the local-gadget tool.
+  Description of the IG tool.
```

Comment:
The same here. I'm wondering if we should always use lowercase `ig` because we sometimes use IG to refer to the one for Kubernetes or the whole project.

cc @mauriciovasquezbernal 

```suggestion
  Description of the ig tool.
```

Reply:
Yes, I think we should call it with lowercase everywhere to make sure it refers to the CLI tool.